frontend/src/components/table/schedule/schedule-parts/base-cell.component.tsx

```diff
@@ -6,8 +6,9 @@ import { CellOptions, CellState } from "./cell-options.model";
 export function BaseCellComponent({
   value,
   className = "",
+  dayType = "",
   isEditable = true,
-  onDataChange,
+  onDataChanged,
   onStateChange,
```

Comment:
@pectom 
Jak będziesz implementować modal do edycji to myślę, że musisz zacząć od tego componentu i dodać do jego opcji callback w stylu `onRightKeyClicked` i potem propagować go do góry. 
Jeszcze jedna rzecz na którą warto zwrócić uwagę - teraz jest dwa componenty które odpowiadają za komórki: ten bazowy i jeden który dziedziczy po nim i odpowiada za komórki w harmonogramie. Więc jak będziesz dodawać callback do opcji, to nie zapomnij o tym, żeby przekazać go też tam.

Reply:
Oki, zapamiętam. Dzięki za hinta :D